users/models.py

```diff
@@ -12,6 +14,13 @@ class JustfixUser(AbstractUser):
         help_text="A U.S. phone number without parentheses or hyphens, e.g. \"5551234567\"."
     )
 
+    full_name = models.CharField(
+        'Full name',
+        max_length=FULL_NAME_MAXLEN,
+        blank=True,
+        help_text="The user's full name."
+    )

```

Comment:
The Django `User` model has `first_name` and `last_name` fields.  I was thinking of using `last_name` (it has a much longer character limit) but I thought it'd end up being confusing, so I decided to just add our own `full_name` field.